snmp/datadog_checks/snmp/data/conf.yaml.example

```diff
@@ -16,6 +16,18 @@ init_config:
   #
   # ignore_nonincreasing_oid: false
 
+  ## @param oid_batch_size - integer - optional - default: 10
+  ## Number of OIDs handled by each batch. Performance can be improved by increasing this number, though
+  ## it will also use more resource.
```

Comment:
```suggestion
  ## it will also use more resources.
```

Performance = autodiscovery time?
resources = cpu & memory or only memory?

Reply:
BTW where is this config used?

Reply:
Performance is check run time. Resources is meant to stay vague, but I'd say mostly memory.

Reply:
And it's used in the check, but it wasn't documented.